### DISCUSSION TITLE
high number of temp folders "chrome_BITS_xxxx_xxxxxxxxx

### DISCUSSION THREAD

**Discussion by dornech** [GCuser99/SeleniumVBA#125] [General]
Hi there, compared to SeleniumBasic I recognize a higher number of temp folders being created. I think in the past one folder was created per session. Now several folders are opened, probably a temp folder per webdriver call. Any ideas? 
I mean, it does not do too much harm because I delete such folders via a more general cleanlup commandlet - but it is not "nice" anyway.

**Comment by GCuser99** [comment]
I'm on Windows 11 and I can't replicate. I've looked at "C:\Users\[my user name]\AppData\Local\Temp" and cannot see many temp folders created with that name (actually, only one created a week ago, not associated with tests that I ran today). I'm travelling so can't try on my Windows 10 machine. Maybe someone else confirm whether they have found same?

**Comment by dornech** [comment]
Basically these folders are said to be remainders from Google update. However, I wonder why Google seems to increase update frequency.

**Comment by GCuser99** [comment]
You mean remainders from Chrome browser updates? If so I don't know the answer but seems likely to me that it is unrelated to SeleniumVBA. But let us know if you discover something different...

**Comment by dornech** [comment]
Seems so and this is what you find when doing some research int he internet. I thought probably an (hidden) parameter in the profile/option migth influence this.
